webclient/package.json

```diff
@@ -30,8 +31,8 @@
     "typescript": "^4.4.4"
   },
   "scripts": {
-    "postinstall": "run-script-os",
-    "postinstall:windows": "powershell .\\copy_shared_files.ps1",
+    "postinstall": "run-script-os && npm run prepare",
```

Comment:
I think the code to run whats in `prepare` should probably just go in the `copy_shared_files` (maybe change the name to something more relevant to the extended usage). We use `run-script-os` to run the prepare scripts that are tailored for each specific environment.